### DISCUSSION TITLE
Max ps3 game listing?

### DISCUSSION THREAD

**Discussion by ZTM2K4** [aldostools/webMAN-MOD#516] [Q&A] [answered]
Might be a stupid question but cant seem to find the answer anywhere but would like to know what the max no of games to list is please many thanks :)

**Comment by aldostools** [comment, marked as answer]
The max number of games depends of the free memory available at the moment of the game scanning.
It also depends of the type of game and the memory usage profile selected in /setup.ps3

So there is not a specific number, due it varies in function of other variables.

This is a table of the memory assigned to each game type. Each 64KB is approximately 100 games.
![image](https://user-images.githubusercontent.com/974897/114400302-d01a6d00-9b6f-11eb-8a91-60169473edc0.png)
